src/app/game/game-intermediate-result/game-drawing-display/game-drawing-display.component.html

```diff
@@ -0,0 +1,4 @@
+<div [class]="['user-drawing-board', getFeedbackStyle()]">
+  <img class="user-drawing" [src]="drawingURL" alt="the users drawing" />

```

Comment:
alt="the user's drawing "

Extremely nitty gritty hahah sorry

Reply:
Fixed haha